README.md

```diff
@@ -10,6 +10,10 @@ Use safemode to make areas be safe zones; otherwise they are killzones by defaul
 	* safemode = true --> PvP Control areas are safe zones
 	* default is `false`, making PvP Control areas kill zones
 
+* `pvp_areas.monsterDoNoDamage`
```

Comment:
This would be better as `pvp_areas.monsters_do_not_damage`, to be consistent with overall Minetest style